decidim-core/lib/decidim/webpacker/tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+// GEM_PATH is the shell output of: gem env gemdir
+module.exports = {
+  content: ["./app/**/*.{html,erb,js,rb}", `${GEM_PATH}/**/decidim-*/app/**/*.{html,erb,js,rb}`],
```

Comment:
We should do something with this syntax. Currently GEM_PATH is not defined, therefore, `bin/webpack` exits with error. Options:

- Change to ${process.env.GEM_PATH} and run `GEM_PATH=$(gem env gemdir) bin/webpack`
- Comment it out in the meantime
- Because this file is a template, the gem_path will be replaced with the actual content of `gem env gemdir` once the development_app is generated

cc/ @ferblape

Reply:
This won't work in the development_app, since their Gemfile points to their parent:
```ruby
gem "decidim", path: ".."
gem "decidim-conferences", path: ".."
gem "decidim-consultations", path: ".."
```
Therefore, the purge won't find any changes you do during the development phase, because in your GEM_PATH will be the **origin/develop** version of the gems. We would need something like `decidim-generators/lib/decidim/generators/component_templates/Gemfile.erb` in order to create the proper tailwind.config.js: 

In the development_app should generate this:
```js
content: [
  "./app/**/*.{html,erb,js,rb}",
  "../decidim-*/app/**/*.{html,erb,js,rb}"
],
```
However, in a normal app, whose Gemfile looks like:
```ruby
gem 'decidim', git: REPO, branch: BRANCH
gem 'decidim-conferences', git: REPO, branch: BRANCH
gem 'decidim-consultations', git: REPO, branch: BRANCH
```
their tailwind.config.js will be something like:
```js
content: [
  "./app/**/*.{html,erb,js,rb}",
  `${process.env.GEM_PATH}/**/decidim-*/**/app/**/*.{html,erb,js,rb}`
],
```

cc/ @entantoencuanto